connectivity-proxy/README.md

```diff
@@ -1,148 +1,156 @@
+- Using a curl Pod to call the sample Node.js application via the Connectivity Proxy.
 
-- Within your global account…
-- Choose **Entitlements -> Entity Assignments**
-- Search **for Connectivity Service**
-- Choose **connectivity_proxy** for the plan
-- Choose **Add 1 Service Plan**
-- Choose **Save**
+![Connectivity Proxy for Kubernetes](./assets/connectivity-proxy.drawio.svg)
 
-### Provision the Service in the Kyma Runtime
+## Prerequisites
 
-The next step involves creating a service instance of the Connectivity Proxy and a service binding. Once the creation of the sevice binding is detected by the Kyma Control Plane, the Connectivity Proxy will be provisioned in the runtime into the namespace **kyma-system**. From within the Kyma runtime it will be accessible using the URL **connectivity-proxy.kyma-system.svc.cluster.local:20003**
+- SAP BTP, Kyma runtime instance
+- [curl](https://curl.se/)
+- [Kubernetes tooling](../prerequisites/README.md#kubernetes)
+- The service plan **connectivity_proxy** of the **connectivity** service is assigned to your subaccount as an entitlement. For more information, see [Configure Entitlements and Quotas for Subaccounts](https://help.sap.com/docs/btp/sap-business-technology-platform/configure-entitlements-and-quotas-for-subaccounts?version=Cloud).
 
-- Within your desired namespace enable Istio if it is not already enabled.
+   > [!NOTE]
+   > For subaccounts created after February 15, 2024, this entitlement is assigned automatically.
 
-```shell
-kubectl label namespaces <your namespace> istio-injection=enabled
-```
+- You have the Istio, SAP BTP Operator, Serverless, and Connectivity Proxy modules added. For more information, see [Adding and Deleting a Kyma Module](https://help.sap.com/docs/btp/sap-business-technology-platform/enable-and-disable-kyma-module?version=Cloud).
 
-- Create the Connectivity Proxy Service Instance
+## Procedure
 
-```shell
-kubectl apply -f ./k8s/connectivity-proxy-instance.yaml -n <your namespace>
-```
+### Provisioning Connectivity Proxy in the Kyma Runtime
 
-The Kyma Control Plane should now provision the Connectivity Proxy. This will generate the pod **connectivity-proxy-0** which can be found by running.
+1. Export the environment variable.
 
-```shell
-kubectl get pods -n kyma-system
-```
+   ```bash
+   export NS=<your namespace> # e.g. default
+   ```
 
-### Configuration of the Sample Application
+2. Enable Istio sidecar proxy injection in your namespace.
 
-To validate the setup, start the sample application included in the repo and configure it within the SAP Cloud Connector.
+   ```shell
+   kubectl label namespaces ${NS} istio-injection=enabled
+   ```
 
-#### Starting the localmock application
+3. Create a service instance and a service binding of the Connectivity Proxy service. Once the service instance and the service binding are detected by the Kyma Control Plane, the Connectivity Proxy service is provisioned in the **kyma-system** namespace.
```

Comment:
this is not required as these steps are done via enabling the connectivity proxy module now